nym-api/src/main.rs

```diff
@@ -107,7 +112,11 @@ async fn start_nym_api_tasks(config: Config) -> anyhow::Result<ShutdownHandles>
     let nym_contract_cache_state = rocket.state::<NymContractCache>().unwrap();
     let node_status_cache_state = rocket.state::<NodeStatusCache>().unwrap();
     let circulating_supply_cache_state = rocket.state::<CirculatingSupplyCache>().unwrap();
-    let maybe_storage = rocket.state::<NymApiStorage>();
+    let storage = if let Some(storage) = rocket.state::<NymApiStorage>() {
```

Comment:
I think that thing can backfire in the future. can we just ensure that the storage always exists when setting up the rocket? though I guess since ultimately we'll remove the rocket, maybe it's fine as the stepping stone

Reply:
I mean will we not remove rocket anyway as part of merging this into develop? Seems like why bother with this then?